pulsar-transformations/src/main/java/com/datastax/oss/pulsar/functions/transforms/TransformFunction.java

```diff
@@ -79,7 +79,7 @@
  *     implementing the most common basic transformations</a>
  */
 @Slf4j
-public class TransformFunction implements Function<GenericObject, Void>, TransformStep {
+public class TransformFunction implements Function<GenericObject, Record<Object>>, TransformStep {
```

Comment:
Is it OK to return Record<Object> or should it be Record<GenericObject> ?

Reply:
@eolivelli

Reply:
Now returns `Record<GenericObject>`. Otherwise the PulsarSink starts a producer with Schema JSON.
We slip through Java generic type erasure to pass other types when we have to.